setup.cfg

```diff
@@ -174,3 +174,9 @@ ignore_missing_imports = true
 
 [mypy-redis.sentinel]
 ignore_missing_imports = true
+
+[testenv:changelog]
+basepython = python3.8
+deps = towncrier<21.3
```

Comment:
why < 21.3? If there's a good reason do you mind putting your response in a comment so it's known for future editors?